front/components/assistant_builder/ActionScreen.tsx

```diff
@@ -547,38 +548,45 @@ export default function ActionScreen({
           />
         </ActionModeSection>
 
-        <ActionModeSection show={action?.type === "DUST_APP_RUN"}>
-          <ActionDustAppRun
-            owner={owner}
-            actionConfigration={
-              action?.type === "DUST_APP_RUN" ? action.configuration : null
-            }
-            dustApps={dustApps}
-            updateAction={(setNewAction) => {
-              setBuilderState((state) => {
-                const previousAction = state.actions[0];
-                if (!previousAction || previousAction.type !== "DUST_APP_RUN") {
-                  // Unreachable
-                  return state;
-                }
-                const newActionConfig = setNewAction(
-                  previousAction.configuration
-                );
-                const newAction: AssistantBuilderActionConfiguration = {
-                  type: "DUST_APP_RUN",
-                  configuration: newActionConfig,
-                  name: previousAction.name,
-                  description: previousAction.description,
-                };
-                return {
-                  ...state,
-                  actions: removeNulls([newAction]),
-                };
-              });
-            }}
-            setEdited={setEdited}
-          />
-        </ActionModeSection>
+        {action?.type === "DUST_APP_RUN" && (
```

Comment:
Why do we need this change ? this is legacy action screen

Reply:
We don't need a specific description / title for single actions

Reply:
The change was required because I modified what we expect for  `updateAction` on <ActionDustAppRun />` !